src/pages/FirebaseTest.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import fb from '../lib/firebase';
+import firebase from 'firebase/app';
+
+export default function FirebaseTesting (props) {
```

Comment:
**Suggestion**: Since this component is for testing, it can totally be changed later, but I think it's worth looking up the ES6 syntax for arrow functions! 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
Changed to ES6 syntax in commit 1a220e5